core/templates/components/concept-card/concept-card.directive.ts

```diff
@@ -73,6 +73,9 @@ angular.module('oppia').directive('conceptCard', [function() {
             }
             // TODO(#8521): Remove when this directive is migrated to Angular.
             $rootScope.$apply();
+          }, function(errorResponse) {
+            ctrl.loadingMessage = 'Oops, it looks like this skill has been ' +
+              'deleted. A message has been sent to the lesson creator';
```

Comment:
```suggestion
            ctrl.loadingMessage = 'Oops, it looks like this skill has been ' +
              'deleted.';
```

Reply:
Since we haven't handled "sending of email" in this PR, let's drop that part in the message.

Reply:
done.